release.json

```diff
@@ -3,6 +3,6 @@
   "operator": "0.11.0",
   "version-upgrade-hook": "1.0.9",
   "readiness-probe": "1.0.20",
-  "agent": "107.0.7.8596-1",
+  "agent": "108.0.0.8694-1",
   "agent-tools-version": "100.9.4"
```

Comment:
do we need to bump tools version as well?

Reply:
Yes, I'll update it for symmetry. But it's no longer used as we build the Agent in MEKO now.